+nosnoc/+discrete_time_problem/Stewart.m

```diff
@@ -94,7 +114,8 @@ function create_variables(obj)
             obj.w.x(0,0,opts.n_s) = {{['x_0'], dims.n_x}, model.x0, model.x0, model.x0};
             obj.w.z(0,0,opts.n_s) = {{'z', dims.n_z}, model.lbz, model.ubz, model.z0};
             obj.w.lambda(0,0,opts.n_s) = {{['lambda'], dims.n_lambda},0,inf, 1};
-            obj.w.mu(0,0,opts.n_s) = {{'mu', dims.n_mu},0,inf};
+            obj.w.theta(0,0,opts.n_s) = {{['theta'], dims.n_theta},0, 1, 1/dims.n_theta};
```

Comment:
is the upper bound on theta needed? will again lead to two identical columns in the Jacobian if 
a single theta_i = 1, 
as we have also \sum theta_i = 1